sql/sql_parse.h

```diff
@@ -129,16 +129,22 @@ extern uint sql_command_flags[];
 extern const LEX_CSTRING command_name[];
 
 #ifdef WITH_WSREP
+#include <service_wsrep.h>
```

Comment:
Not a strong opinion.
```suggestion
#include "service_wsrep.h"
```

Reply:
fixed